examples/complete/variables.tf

```diff
@@ -103,6 +103,24 @@ variable "arn_format" {
   default     = "aws"
 }
 
+variable "create_dynamodb_table" {
+  type        = bool
+  description = "Boolean for if a DynamoDB table is to be created for batch locking."
+  default     = true
```

Comment:
can a user opt out of creating a table to disable this feature?

Reply:
Added another variable 'batch_locking_enabled', but will iterate on it to see if this can be improved.

Reply:
it might be a little more code but it would be nice to let the user customize it with just this one `bool` otherwise the UX becomes more unwieldy and we need to review 2*2*2 cases of `batch_locking_enabled` `create_dynamodb_table` and `user_managed_dynamodb_table_name` values being changed from default.

Reply:
Makes sense. Added conditions that check— 

1. If `create_dynamodb_table` is `false` and `user_managed_dynamodb_table_name` is `null`, then no resources should be created. `dynamodb_table_name` passed to the lambda will be `null`, so the batch locking will be avoided.
2. If both of the values are truthy, the one provided by the user is used.
3. If either one of them is truthy, then that value will be used.